golang/client.go

```diff
@@ -436,7 +436,6 @@ func (cli *defaultClient) doHeartbeat(target string, request *v2.HeartbeatReques
 			Message: resp.GetStatus().GetMessage(),
 		}
 	}
-	cli.log.Infof("send heartbeat successfully, endpoints=%v", endpoints)
```

Comment:
Change log level to debug instead of removal?

Reply:
Consider changing to debug level